hs/Scripts.hs

```diff
@@ -89,8 +90,14 @@ postScripts = sequence_ (map toScript [
                                           "/static/js/graph/objects/node.js",
                                           "/static/js/post/fill_textboxes.js",
                                           "/static/js/graph/create_data.js",
-                                          "/static/js/post/update_categories.js"])
-
+                                          "/static/js/post/update_categories.js",
+                                          "static/js/common/course_description.js",
+                                          "/static/js/common/objects/course.js",
+                                          "/static/js/common/objects/section.js",
+                                          "/static/js/common/utilities/util.js",
+                                          "https://cdnjs.cloudflare.com/ajax/libs/react/0.13.1/react.js",
+         "https://cdnjs.cloudflare.com/ajax/libs/react/0.13.1/JSXTransformer.js"])
```

Comment:
I've just added some global javascript includes. Please remove the two React scripts here; they should be automatically loaded.

Note: you'll need to do a pull from `upstream/react-post` to get the updates.